swarmcg/utils.py

```diff
@@ -0,0 +1,27 @@
+import contextlib
+import random
+import sys
+
+import MDAnalysis as mda
+
+
```

Comment:
The reason why I was using this, is that FST-PSO has his own output that I don't want to show to the users (we still acknowledge them correctly in the header displayed at program execution). We do the necessary checks for input consistency + progress/metrics informations ourselves, so we really don't care about FST-PSO error messages or progress informations. I'm not sure this is the correct way to handle the IO for this kind of things though.

Reply:
will figure it out, I guess